tls/make-certs.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+set -xe
+
+docker build -t temporal_cfssl:test - < $(PWD)/tls/Dockerfile.cfssl
```

Comment:
I think you mean `$PWD` or `${PWD}` instead of `$(PWD)`?

Reply:
Yeah, I did.